astropy/coordinates/representation.py

```diff
@@ -1579,14 +1579,20 @@ def from_cartesian(cls, cart):
     def represent_as(self, other_class, differential_class=None):
         # Take a short cut if the other class is a spherical representation
 
-        # TODO: this could be optimized to shortcut even if a differential_class
-        # is passed in, using the ._re_represent_differentials() method
+        # TODO: implement here after #11469
```

Comment:
A quick note to say that you can rebase your branch to @mhvk's branch used in #11469 - if you want to talk through how to do that, ping me on slack!

Reply:
Thanks! I definitely would if it were command-line, but I've been using [Tower](https://www.git-tower.com/mac) lately and that thing has made my git life so much better ([at]tower can I get a promotional discount 😆).